hoot-core/src/main/cpp/hoot/core/conflate/phone/PhoneNumberLocator.h

```diff
@@ -37,7 +37,6 @@
 
 // libphonenumber
 #include <phonenumbers/geocoding/phonenumber_offline_geocoder.h>
-using namespace i18n::phonenumbers;
```

Comment:
Removed `using namespace` directives from header files so that they don't propagate outside of the intended source.